server/__init__.py

```diff
@@ -0,0 +1 @@
+
```

Comment:
I don't think we want to make pulp.server a python package itself. The entry point loads underneath it so this should be here. I think this file should be deleted.